product_variant_supplierinfo/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Product supplier info per variant',
+    'summary': 'Supplier info to product variant scope',
+    'version': '8.0.1.0.0',
+    'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, '
```

Comment:
For sure
note : as I have done a lot of change, can I add akretion as a co-author? (if not no problem)